ddd-koans/Value Object Pt. 2 & Entity/Combinable/task-info.yaml

```diff
@@ -0,0 +1,51 @@
+type: edu
+files:
+- name: src/Main.kt
+  visible: true
+  placeholders:
+  - offset: 227
+    length: 127
+    placeholder_text: TODO("How should the length of notes be calculated?")
```

Comment:
doesn't look like a part of this task

Reply:
or maybe this is becuase of dependency :thinking:

Reply:
Yeah, this is due to being dependency.